src/getting-started/create-bootable-media/bootable-usb-using-macos.md

```diff
@@ -1 +1,99 @@
-Users creating installation media from MacOS can use Etcher. Etcher is an open source startup disk creator capable of writing ISO images to portable storage media.
\ No newline at end of file
+# Create a Bootable Pop!\_OS USB in macOS
+
+This section explains the process of creating a bootable Pop!_OS USB in macOS using balena Etcher.
+
+- [Install ISO flashing software.](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#install-etcher)
+- [Download the Pop!\_OS.iso file.](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#download-the-pop_osiso-file)
+- [Verify the Pop!\_OS.iso Hash Value](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#verify-the-hash-value)
+- [Flash the ISO to your USB storage device.](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#create-the-bootable-usb)
+
+---
+
+## Install Etcher
+
+1. Navigate to Etcher's [download page](https://www.balena.io/etcher/) and click `Download for macOS`.
+
+![Download Etcher](/images/create-bootable-usb-macos/download-etcher.png)
+
+2. Click the download folder from the Dock and select the Etcher .dmg file.
```

Comment:
It's called the Downloads folder (capital D and plural), documented e.g. here: https://support.apple.com/guide/mac-help/dock-mh35859/mac